### PR TITLE
[ContactStructuralMechanicsapplication] Adding multiple search validation tests

### PR DESCRIPTION
**Pull request by loumalouomega**
The test was missing

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_3D/3D_multi_contact_test_materials.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "properties" : [{
+        "model_part_name" : "Parts_Lower",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "KratosMultiphysics.StructuralMechanicsApplication.LinearElastic3DLaw"
```
Comment (philbucher) [inline applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_3D/3D_multi_contact_test_materials.json:7]: `LinearElastic3DLaw` is enough, the rest is ignored anyway...

**Review by philbucher — APPROVED** [review]
guess its fine if I approve, since its just a test

Don't know the details though

File: applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_3D/3D_multi_contact_test_materials.json (hunk 1 above)
Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_3D/3D_multi_contact_test_materials.json:7]: Sure, @jginternational can we change this?

**Comment by philbucher** [comment]
I made an issue some time ago already

**Pull request merged**